proposed/2021/VisualStudioPackageInstallDeepLink.md

```diff
@@ -0,0 +1,163 @@
+# Visual Studio Package Install Deep Link
+
+- Author Name: Chris Gill ([chgill-MSFT](https://github.com/chgill-MSFT))
+- Start Date: 07/19/2021
+- GitHub Issue: https://github.com/NuGet/Home/issues/11032
+- GitHub PR: N/A
+
+## Summary
+
+Support deep links for NuGet.org packages that, when clicked, open the package in the Visual Studio NuGet Package Manager UI (PMUI) for a 1 click install experience.
+
+Applications:
+
+* Add “Install with Visual Studio” link to package README, documentation, or tutorials
```

Comment:
UX nit: The suggestion here is 'Install with Visual Studio' however looking at the scenarios the truth is 'launch PMUI with pre-selected search/details' and the user still has to click 'install' at some point.  Just making sure we have truth in advertising.  Consider testing with "Open in Visual Studio"?  You provide an example of VS Code/Dracula -- I can't recall if they use similar language, but the pattern is similar...the vscode:// protocol opens the detail page and still requires the user to click Install -- so follow similar language there whatever the original entry point is for that deep link.

Reply:
This is a good point!